### PR TITLE
Install script for Raspberry Pi

### PR DESCRIPTION
**Pull request by dhylands**
This includes the systemd script, along with the init.d script for setting up the iptables port remapping.

### CODE REVIEW

**Comment by dhylands** [comment]
@hobinjk These are the startup files that I used on the image we released with one minor tweak. It uses npm start rather than ./app.js

**Comment by andrenatal** [comment]
I'd suggest for use to use journalctl to handle the logs. I think is better for rotation, manipulation, storing plus few other factors.

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```
Comment (hobinjk) [inline install.sh]: This file should have the execute bit set

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    git clone https://github.com/moziot/gateway.git
+fi
+cd gateway
+npm install .
+
+# Setup systemctl so that the gateway will start at boot time
+sudo cp systemd/system/mozilla-iot-gateway.service /etc/systemd/system
+sudo systemctl enable mozilla-iot-gateway.service
+
+sudo cp gateway-iptables.sh /etc/init.d
```
Comment (hobinjk) [inline install.sh]: `gateway-iptables.sh` should be `etc/init.d/gateway-iptables.sh`

Comment (dhylands) [inline install.sh]: Heh - yeah - I have that change in my uncommitted changes.

I've also commented out creating the self-signed cert.

File: install.sh (hunk 1 above)
Comment (dhylands) [inline install.sh]: Yes - will change

File: run-app.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+run_app() {
```
Comment (lightsofapollo) [inline run-app.sh]: Just FYI I will probably put the A/B setup in this file after this lands... Let me know if that makes sense.

Comment (dhylands) [inline run-app.sh]: Yeah - I think that makes sense.

**Pull request merged**